frontend/src/routes/Applications/AdvancedConfiguration.tsx

```diff
@@ -240,6 +240,12 @@ export default function AdvancedConfiguration() {
         // edit
         actions.push({
             id: `edit${kind}`,
+            /*
+                t('Edit subscription')
+                t('Edit channel')
+                t('Edit placement')
+                t('Edit placement rule')
+                */
```

Comment:
When template strings or variables are passed into the t function, the parser cannot find these keys. Adding a comment makes sure the keys get put into the translation file.